test/unit/events.spec.ts

```diff
@@ -1,16 +1,15 @@
-import { SinonStatic, SinonFakeTimers } from "sinon";
-
+import { SinonStatic } from "sinon";
 import Flicking from "../../src/Flicking";
 import { EVENTS, DIRECTION } from "../../src/consts";
 import { FlickingEvent, NeedPanelEvent, FlickingOptions } from "../../src/types";
 import { horizontal, vertical } from "./assets/fixture";
-import { createFlicking, createHorizontalElement, cleanup, simulate, waitFor, waitEvent } from "./assets/utils";
+import { createFlicking, createHorizontalElement, cleanup, simulate, waitFor, waitEvent, tick } from "./assets/utils";
 import { merge } from "../../src/utils";
 import Viewport from "../../src/components/Viewport";
 
 declare var sinon: SinonStatic;
 
-describe("Events", () => {
+describe.only("Events", () => {
```

Comment:
@WoodNeck 

Remove `only`.

Reply:
Wrong commit (5b311cb) was between commits, I've fixed it.